.github/workflows/ci.yml

```diff
@@ -135,7 +135,7 @@ jobs:
       - name: Run Unit Tests
         working-directory: ${{ github.workspace }}
         run: |
-          tests/unit_tests/simpll/runTests
+          build/tests/unit_tests/simpll/runTests
```

Comment:
I guess it could be "simplified" to just `ninja test` if working directory would be changed to `${{ github.workspace }}/build`.

Reply:
Good idea, will do that.